### DISCUSSION TITLE
How do I properly configure styles?

### DISCUSSION THREAD

**Discussion by mirmalag** [mariuz/flamerobin#357] [Q&A] [answered]
New styles are driving me nuts.
Maybe it's me, but I don't know ho to get the wished effect.
I'm sure that the moment I understand the rationale behind theme, language and style and how they relate to each other all will be brilliantly clear :-)
Can somebody please explain or point me to a useful resource?

Thank you,
Mirco

**Comment by Jdochoa** [comment, marked as answer]
Please read Scintilla documentation.
https://www.scintilla.org/
Or notepad++
https://npp-user-manual.org/docs/themes/
